tests/test_models.py

```diff
@@ -90,6 +90,42 @@ def test_string_representation(self):
         assert expected_repr == course_skill.__repr__()
 
 
+@mark.django_db
+class TestXBlockSkills(TestCase):
+    """
+    Tests for the ``XBlockSkills`` model.
+    """
+
+    def test_xblock_skill_string_representation(self):
```

Comment:
```suggestion
    def test_string_representation(self):
```
This is already in `TestXBlockSkills` class, so we can eliminate redundant naming.